src/main/java/com/mineboundteam/minebound/registry/RecipeRegistry.java

```diff
@@ -0,0 +1,20 @@
+package com.mineboundteam.minebound.registry;
+
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.RegistryObject;
+
+import static com.mineboundteam.minebound.MineBound.MOD_ID;
+
+import com.mineboundteam.minebound.crafting.AlloyFurnaceRecipe;
+import com.mineboundteam.minebound.crafting.serializers.AlloyFurnaceSerializer;
+
+public class RecipeRegistry {
+    public static final DeferredRegister<RecipeSerializer<?>> RECIPE_SERIALIZERS = DeferredRegister.create(ForgeRegistries.RECIPE_SERIALIZERS, MOD_ID);
+    public static final RegistryObject<RecipeSerializer<AlloyFurnaceRecipe>> ALLOY_FURNACE_SERIALIZER = RECIPE_SERIALIZERS.register("alloy_furnace", () -> new AlloyFurnaceSerializer<>(200));
+
+    public static final RecipeType<AlloyFurnaceRecipe> ALLOY_FURNACE_RECIPE = RecipeType.register("alloy_furnace");
```

Comment:
When attempting to launch the game, an error caused by this line is thrown. Below are the relevant details:

```
Exception message: java.lang.IllegalStateException: Registry is already frozen (trying to add key ResourceKey[minecraft:recipe_type / minecraft:alloy_furnace])
	at com.mineboundteam.minebound.registry.RecipeRegistry.<clinit>(RecipeRegistry.java:18) ~[%2384!/:?] {re:classloading}
	at com.mineboundteam.minebound.registry.Registry.RegisterMod(Registry.java:13) ~[%2384!/:?] {re:classloading}
	at com.mineboundteam.minebound.MineBound.<init>(MineBound.java:39) ~[%2384!/:?] {re:classloading}
```

Reply:
Based on some very limited research, line 18 may not even be necessary. But take that with a grain of salt, as I am not confident in that claim.